Header header          #standard header information
geometry_msgs/Pose sensor_pose_
geometry_msgs/Pose Tnow # Current pose estimate.
float64 distance_moved_in_last_node_

NDTNodeMsg[] nodes
NDTEdgeMsg[] edges
 
#  NDTFeatureGraph::Params params_;
#  NDTFeatureFuserHMT::Params fuser_params_;
#  semrob_generic::MotionModel2d::Params motion_params_;

#  pcl::PointCloud<pcl::PointXYZ> pointcloud_vis;

